documentation/src/components/PageNavigation.astro

```diff
@@ -0,0 +1,97 @@
+---
+import Show from 'src/components/Show.astro'
+import { getPathParamsFromId } from 'src/languages'
+import { getLocalizedSidebar } from 'src/navigation'
+import { getPrevNext } from 'src/navigation'
+import {PAGE_NAVIGATION} from 'src/consts'
+
+const { slug, lang } = getPathParamsFromId(Astro.url.pathname);
+const baseSidebar = await getLocalizedSidebar(slug, lang);
+
+const { prevPage, nextPage } = getPrevNext(baseSidebar, Astro.url.pathname);
+---
+
+<nav class="pagination-links">
```

Comment:
```suggestion
<nav class="pagination-links" data-pagefind-ignore="all">
```

https://pagefind.app/docs/indexing/#removing-individual-elements-from-the-index

Reply:
resolved!